easybuild/easyconfigs/a/ant/ant-1.10.5-Java-1.8.eb

```diff
@@ -1,24 +1,29 @@
-easyblock = 'PackedBinary'
-
 name = 'ant'
 version = '1.10.5'
 versionsuffix = '-Java-%(javaver)s'
 
-homepage = 'http://ant.apache.org/'
+homepage = 'https://ant.apache.org/'
 description = """Apache Ant is a Java library and command-line tool whose mission is to drive processes described in
  build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of
  Java applications."""
 
 toolchain = SYSTEM
 
-source_urls = ['http://apache.cu.be//ant/binaries']
-sources = ['apache-%(name)s-%(version)s-bin.tar.gz']
-checksums = ['cd52c6f77a0b1de7174addc94e5a512d86978ac9173a30cd11f4ab26f6d8deba']
+source_urls = ['https://archive.apache.org/dist/%(name)s/source/']
+sources = ['apache-%(name)s-%(version)s-src.tar.gz']
```

Comment:
Why is this changed to `-src.tar.gz` while it's left to `-bin.tar.gz` for `ant` 1.10.6 & 1.10.7?

Reply:
Because for 10.6+ there still is a -bin package

Reply:
OK, thanks for clarifying!